archunit-junit/junit5/engine/src/test/java/com/tngtech/archunit/junit/internal/ArchUnitTestEngineTest.java

```diff
@@ -116,6 +118,7 @@
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.WARN)
```

Comment:
As we only have one single usage of `MockitoExtension`, I wonder whether we shouldn't just call `MockitoAnnotations.initMocks(testInstance);` directly (and drop the `mockito-junit-jupiter` dependency again).

Reply:
Hmm, yeah, sounds fair :thinking: Seems overkill to have an extra dependency for that and also not really saving anything in this case :grimacing:

Reply:
On second thought, `initMocks(..)` is meanwhile deprecated, you should now call `openMocks(..)` and hold the returned `AutoCloseable` to close it in the tear down method :zany_face:
Once we migrate the other code from JUnit 4 to JUnit 5 we'll also need this in more places, so let's just keep the dependency for now...